src/common/utils/kaltura-params.js

```diff
@@ -59,23 +64,26 @@ function setSessionId(playerConfig: PartialKPOptionsObject, sessionId: string):
 }
 
 /**
- * @param {PKMediaSourceObject} source - PKMediaSourceObject
+ * @param {string} url - url
  * @param {string} sessionId - session id
- * @return {void}
+ * @param {string} paramName - optional param name of the session id
+ * @return {string} - the url with the new sessionId
  * @private
  */
-function updateSessionIdInUrl(source: Object = {}, sessionId: ?string): void {
+function updateSessionIdInUrl(url: string, sessionId: ?string, paramName: ?string): string {
+  paramName = paramName || PLAY_SESSION_ID;
```

Comment:
either make it mandatory or not, doesn't make a lot of sense to make it optional

Reply:
fixed - please see i understood you correctly